tests/queries/0_stateless/03304_strncmp.sql

```diff
@@ -0,0 +1,50 @@
+-- {echoOn }
```

Comment:
"Negative" tests for error cases are missing:
- what if too few arguments are passed?
- what if too many arguments are passed?
- what if arguments are of an incorrect type?
- etc.

Use syntax `{ serverErrror ... }` to test.

Think carefully about what could possibly go wrong.